golang.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt" //here is a line coment
+
+// FileGetter is usd to retrieve de code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
```

Comment:
Typo inside comment in token 'returs'. Possible corrections:return (78),returns (63),returs (1)